src/main/java/org/oasis_eu/portal/front/my/ContactAJAXServices.java

```diff
@@ -44,41 +51,80 @@
     @Value("${mail.contact.to}")
     private String mailContactTo;
 
-    @RequestMapping(value = "/send", method = RequestMethod.POST)
-    public void send(@RequestBody @Valid ContactRequest contactRequest, Locale locale) throws MessagingException {
-        logger.debug("send {}'s contact type with subject {}, body {}", contactRequest.motive, contactRequest.subject, contactRequest.body);
+    @Value("${captcha.secret}")
+    private String captchaSecret;
 
-        SimpleMailMessage message = new SimpleMailMessage();
+    @Value("${captcha.sitekey}")
+    private String sitekey;
 
-        UserInfo userInfo = userInfoService.currentUser();
-        String userEmail = userInfo.getEmail();
+    @Value("${captcha.url}")
+    private String captchaUrl;
 
-        message.setTo(mailContactTo);
-        message.setFrom(mailContactFrom);
-        message.setReplyTo(userEmail);
-        if (contactRequest.copyToSender) {
-            message.setCc(userEmail);
+    @RequestMapping(value = "/send", method = RequestMethod.POST)
+    public ResponseEntity<String> send(@RequestBody @Valid ContactRequest contactRequest, Locale locale) throws MessagingException {
+        logger.debug("send {}'s contact type with subject {}, body {}", contactRequest.motive, contactRequest.subject, contactRequest.body);
+
+        HttpHeaders headers = new HttpHeaders();
+        RestTemplate rest = new RestTemplate();
+        headers.add("Content-Type", "application/json");
+        headers.add("Accept", "*/*");
+        HttpEntity<String> requestEntity = new HttpEntity<String>("", headers);
+        UriComponents uriComponents =
+                UriComponentsBuilder.newInstance()
```

Comment:
The newInstance() is not necessary since you the create one fromUriString just below